splunk_add_on_ucc_framework/commands/init.py

```diff
@@ -25,9 +25,13 @@
 
 logger = logging.getLogger("ucc_gen")
 
-ADDON_NAME_RE_STR = r'^[^<>:"/|?*]+$'
+ADDON_NAME_RE_STR = (
```

Comment:
If this is the same as in `schema.json` can we introduce a test to verify it?

A small unit test importing this file and asserting that it's the same as in `schema.json` would do the trick.

Alternatively we can leave a comment here (less preferable from my standpoint).

Reply:
Added the testcase in test_init file.